### ISSUE TITLE
Date Picker looks strange

### ISSUE THREAD

**Issue by rsudev** [cgeo/cgeo#5148]
The other day for the first time since probably a year or so I wanted to change the log date of an entry. The upcoming date picker looks somewhat strange to me, as it does not contain an OK button (and even does not work at all on my device in my version, but that is another story).
Does the date picker look and work as expected for you?

**Comment by kumy** [comment]
Mine looks like:
![screenshot_2015-08-21-23-37-54](https://cloud.githubusercontent.com/assets/176794/9419705/4979b66c-485e-11e5-8793-7c143675a782.png)

No ok button too. I'm okay to say it's quite non intuitive at first.

**Comment by SammysHP** [comment]
Maybe we can start using Material style when we change something here.

https://github.com/prolificinteractive/material-calendarview

**Comment by Bananeweizen** [comment]
It has been like that for me for a long time, I cannot even remember a previous look. I believe this is a system component, isn't it?

@SammysHP before starting to explore Material I suggest to first change the main build to gradle. Even the material related support and compat libs are hard to use with an And/Eclipse based build nowadays.

**Comment by rsudev** [comment]
I have seen that the code is in its current state already for quite some time (somewhen last year), but as my phone OS changed in the meantime (from Kitkat to Lollipop) I wanted to confirm my impression.

My suggestion would be to first switch the current implementation from a DatePicker view to the DatePickerDialog (as per http://developer.android.com/guide/topics/ui/controls/pickers.html).
This gives us the 'OK'-button (and by that a clear and clean interaction for the user) for free (and a resource and some lines of code less).

**Comment by kumy** [comment]
The TimePicker I ve added for GK logging would also need an update.

**Comment by rsudev** [comment]
TimePicker adapted as well...

**Comment by kumy** [comment]
@rsudev I like the new `DatePiker` UI.
